legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/ExecutionNodeSerializerHelper.java

```diff
@@ -157,6 +158,7 @@ private static class Writer implements ISerializationWriter
         {
             this.generator = generator;
             this.includeEnumType = config != null && config.includeEnumType;
+            this.dateTimeFormat = config != null ? config.dateTimeFormat : null;
```

Comment:
we don't ahve similar requirement for strictDate?

Reply:
As of now it is just the DateTime fields as Pure serialization handles this by default, we don't want to default the same on Alloy by adding the offset +0000 because that may potentially break production usecases.